### ISSUE TITLE
resourcedir

### ISSUE THREAD

**Issue by careri** [Starcounter/Starcounter#3095]
Hi!

I can't seem to provide the resource dir when launched from VS.
Debug settings in VS:
Command line arguments:
```
-d=HAD --name=HAD --resourcedir=WebApp
```
Working directory:
```
D:\source\Scdb2Apps\HeadsAdmin\BootApp\bin\Debug
```

It works if I launch from cmd with PWD=D:\source\Scdb2Apps\HeadsAdmin\BootApp\bin\Debug
```
star.exe -d=HAD --name=HAD --resourcedir=WebApp HAD.exe
```

I figured out that the static resources can be viewed at this url:
http://localhost:8080/staticcontentdir

This is when launched from Visual Studion:
```
{"Items":[{"Port":8080,"Folder":"c:\\program files\\starcounter\\clientfiles\\staticfiles"},{"Port":8080,"Folder":"d:\\source\\scdb2apps\\headsadmin\\bootapp\\bin\\debug"}]}
```

This is when launched from CMD (webapp is appended):
```
{"Items":[{"Port":8080,"Folder":"c:\\program files\\starcounter\\clientfiles\\staticfiles"},{"Port":8080,"Folder":"d:\\source\\scdb2apps\\headsadmin\\bootapp\\bin\\debug\\webapp"}]}
```

**Comment by careri** [comment]
Tried providing the full path to the resourcedir form VS, didn't work either. So no debugging from VS for me :(

**Comment by per-samuelsson** [comment]
Hi @careri,

Thanks for reporting!

This worked before, right? If so, my guess is that its related to VS 12 -> 15 migration. But its weird not even the full explicit path work.

I can investigate this when I kick up those other two issues relating to the resource directory, i.e. #2976 and #2898.

**Comment by per-samuelsson** [comment]
@careri 

As a workaround, try specifying the resource dir in the "Working Directory" textbox (`Project | Properties | Debug | Working directory`). Think that should work.

**Comment by careri** [comment]
Morning! Yes, this used to work. But rework to make it more straight forward seems nice.

**Comment by per-samuelsson** [comment]
> [...] I have assemblies and other files that I work with in bin\debug

```
var workingDir = Path.Combine(Application.Current.WorkingDirectory, "..");
```

:sunglasses: 

Nah, just kidding. Lets just wait a little then, until I'm working out #3099. My aim is to have it fixed by tomorrow-ish.

**Comment by careri** [comment]
Yepp, I'll get by for now

**Comment by per-samuelsson** [comment]
Done. Merged and documented as part of #3099.
